conf/data.yml.example

```diff
@@ -16,5 +16,10 @@
 accumuloIndexTable: webindex_search
 # Fluo Application Name
 fluoApp: webindex
+#Webindex builds multiple data sets for its computation.  Each of these data
```

Comment:
Can you add a space between # and the first character?  I find this hard to read.